guides/source/debugging_rails_applications.md

```diff
@@ -628,11 +656,7 @@ method.
 
 ```
 (byebug) next
-
-Next advances to the next line (line 6: `end`), which returns to the next line
-of the caller method:
-
```

Comment:
We are actually explaining that same thing a few lines before, and this is actually `byebug`'s literal output, so the explanation does not really make sense there.